pyproject.toml

```diff
@@ -4,12 +4,12 @@ dynamic = ["version"]  # Load from ml_dtypes.__version__.
 description = ""
 readme = "README.md"
 requires-python = ">=3.9"
-license = {file = "LICENSE"}
+license = "Apache-2.0"
+license-files = ["LICENSE", "LICENSE.eigen"]
 authors = [{name = "ml_dtypes authors", email="ml_dtypes@google.com"}]
 classifiers = [
     "Programming Language :: Python :: 3",
     "Programming Language :: Python :: 3 :: Only",
-    "License :: OSI Approved :: Apache Software License",
```

Comment:
I would still keep this (shows up on the PyPI page of the project)

Reply:
I removed this because `pip install -e .` fails when it is included.